osapi_exporters/osapi_exporter_types/cinderbackend.py

```diff
@@ -0,0 +1,153 @@
+                'overcommit_ratio': None,
+                'free_until_overcommit': None,
+                'reserved_percentage': caps['reserved_percentage']
+            }
+            shard_name = caps['vcenter-shard']
+            logging.debug("Got stats for pool {} ({}/{})".format(
+                self.region,
+                shard_name,
+                caps['volume_backend_name']))
+
+            if caps['total_capacity_gb']:
+                pool_data['overcommit_ratio'] = caps['allocated_capacity_gb'] / caps['total_capacity_gb']
+            else:
+                pool_data['overcommit_ratio'] = 0
+
+            free_until_overcommit = None
+            if caps['total_capacity_gb'] and 'max_over_subscription_ratio' in caps:
+                free_until_overcommit = (
+                    caps['total_capacity_gb'] * float(caps['max_over_subscription_ratio'])
+                    - caps['allocated_capacity_gb'] )
+            pool_data['free_until_overcommit'] = free_until_overcommit
+
+            logging.debug('({}/{})-total_capacity_gb = {}'.format(
+                shard_name, caps['volume_backend_name'],
+                caps['total_capacity_gb']
+            ))
+            logging.debug('({}/{})-free_capacity_gb = {}'.format(
+                shard_name, caps['volume_backend_name'],
+                caps['free_capacity_gb']
+            ))
+            logging.debug('({}/{})-allocated_capacity_gb = {}'.format(
+                shard_name, caps['volume_backend_name'],
+                caps['allocated_capacity_gb']
+            ))
+            logging.debug('({}/{})-free_until_overcommit = {}'.format(
+                shard_name, caps['volume_backend_name'],
+                free_until_overcommit
+            ))
+            logging.debug('({}/{})-max_over_subscription_ratio = {}'.format(
+                shard_name, caps['volume_backend_name'],
+                caps['max_over_subscription_ratio']
+            ))
+            logging.debug('({}/{})-overcommit_ratio = {}'.format(
+                shard_name, caps['volume_backend_name'],
+                pool_data['overcommit_ratio']
+            ))
+            self.metric_count += len(pool_data)
+
+            if shard_name not in self.pool_data:
+                self.pool_data[shard_name] = dict()
+
+            if caps['volume_backend_name'] not in self.pool_data[shard_name]:
+                self.pool_data[shard_name][caps['volume_backend_name']] = dict()
+
+            self.pool_data[shard_name][caps['volume_backend_name']] = pool_data
+
+        logging.debug("Pool data = {}".format(self.pool_data))
+
+    def export(self):
+        """Add the stats and export them."""
+        for shard in self.pool_data.keys():
+            logging.debug("shard = {}".format(shard))
+            for backend in self.pool_data[shard].keys():
+                logging.debug("  backend {}".format(backend))
+
+                self.gauge['total_capacity_gb'].labels(
```

Comment:
Is there automatically a prefix applied? Otherwise, I would suggest adding an `openstack_cinder_` prefix to every metric.